Makefile

```diff
@@ -123,9 +123,9 @@ thinkhazard/static/build/%.css: $(LESS_FILES) .build/node_modules.timestamp
 	.build/venv/bin/pip install -r requirements.txt
 	touch $@
 
-.build/flake8.timestamp: $(PY_FILES)
```

Comment:
As discussed, removing the `$(PY_FILES)` dependency will make this rule check the py files only once. And previously only the modified files were checked. It would be good to keep that.